.github/workflows/build-test-push-workflow.yml

```diff
@@ -185,17 +166,14 @@ jobs:
         uses: actions/setup-python@v2
       - name: Install AWS CLI
         run: |
-          curl "${{ steps.dotenv.outputs.AWSCLI_URL}}" -o "awscli-bundle.zip"
-          unzip awscli-bundle.zip
-          sudo ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
+          curl "${{ steps.dotenv.outputs.AWSCLI_URL}}" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          sudo ./aws/install --update
           aws --version
       - name: Setup Go
         uses: actions/setup-go@v2
         with:
           go-version: ${{ steps.dotenv.outputs.GO_VERSION }}
-      - name: Install Go Lint
```

Comment:
assuming not needed?

Reply:
yup, support for golint is deprecated with the latest versions.